packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -163,19 +198,34 @@ async function downloadSimpleFinTransactions(acctId, since) {
     throw BankSyncError(res.error_type, res.error_code);
   }
 
-  const {
-    transactions: { all },
-    balances,
-    startingBalance,
-  } = res;
+  let accounts = res;
+  if (!batchSync) {
+    accounts = {
+      placeholder: res,
+    };
+  }
 
-  console.log('Response:', res);
+  let retVal = {};
+  for (const [accountId, data] of Object.entries(accounts) as [string, any][]) {
```

Comment:
Can `any` have a tighter type?

Reply:
Fixed now, I've stopped short of typing the entire bank sync transaction response - that'll be tackled when the whole file is looked at as part of the TS migration.